src/status_im/common/home/actions/view.cljs

```diff
@@ -456,11 +465,17 @@
   (let [current-pub-key (rf/sub [:multiaccount/public-key])]
     [quo/action-drawer
      [[(view-profile-entry public-key)
+       (when-not (= current-pub-key public-key)
+         (when config/show-not-implemented-features?
+           (rename-entry)))
```

Comment:
Here's a different example of add more layers of `when` expressions, but in this case I did this because I wanted to avoid using `and` or `or` with the `when-not` expression.